src/EditorFeatures/Core/Implementation/Adornments/GraphicsTag.cs

```diff
@@ -11,28 +12,36 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Adornments
     /// </summary>
     internal abstract class GraphicsTag : ITag
     {
-        protected static SolidColorBrush VerticalRuleBrush;
-        protected static Color VerticalRuleColor;
+        private readonly IEditorFormatMap _editorFormatMap;
+        protected Brush _graphicsTagBrush;
+        protected Color _graphicsTagColor;
+
+        protected GraphicsTag(IEditorFormatMap editorFormatMap)
+        {
+            _editorFormatMap = editorFormatMap;
+        }
 
         protected virtual void Initialize(IWpfTextView view)
         {
-            if (VerticalRuleBrush != null)
+            if (_graphicsTagBrush != null)
             {
                 return;
             }
 
             // TODO: Refresh this when the user changes fonts and colors
 
-            // TODO: Get from resources
-            var lightGray = Color.FromRgb(0xE0, 0xE0, 0xE0);
+            // If we can't get the color for some reason, fall back to a hardcoded value
+            // the editor has for outlining.
+            var lightGray = Color.FromRgb(0xA5, 0xA5, 0xA5);
```

Comment:
Other tags don't try to match their color to an existing editor color.

We're just trying to be resilient in case something goes off the rails.  I don't want to remove this and then have to fight a last minute fire because something is wonky in VS.